array-manipulation-2/src/drop-right.js

```diff
@@ -0,0 +1,5 @@
+/* exported dropRight */
+function dropRight(array, count) {
+  const newArray = array.slice(0, array.length - count);
+  return newArray;
+}
```

Comment:
Please make sure `dropRight` consistently returns an empty array when `count` is greater than `array.length`:

```js
dropRight(['chicken', 'waffle', 'syrup', 'cheese'], 5);
// -> []
```